packages/service-library/src/servicelib/rabbitmq/_utils.py

```diff
@@ -51,7 +56,10 @@ async def wait_till_rabbitmq_responsive(url: str) -> bool:
 
 
 def get_rabbitmq_client_unique_name(base_name: str) -> str:
-    return f"{base_name}_{socket.gethostname()}"
+    random_string = "".join(
+        random.choice(CHARACTERS) for _ in range(_GENERATE_RANDOM_STRING_LENGTH)
+    )
+    return f"{base_name}_{socket.gethostname()}_{random_string}"
```

Comment:
I'm not convinced by this. The whole idea of using the hostname here was to make it unique.

I would rather drop the hostname and use a larger random string, which makes collisions less likely. You might even get away with using a uuid from which you strip the `-`.

Reply:
Please note that all your tests will fail if something fishy is happening with the queue names. You might want to check what your change introduced.